File: py-recitations-solutions/helloworld/helloworld.py
# helloworld.py

#  print your greetings to the world using python
s = "Hello Beautiful from Python!"
print(f"\n{s}\n")

# Bonus : Write to a text file
# create a text file in your folder
f = open("py-recitations-solutions\helloworld\hellopy.txt","w")
f.write(s)
f.close()

f = open("py-recitations-solutions\helloworld\hellopy.txt","r")
print("from text file : ",f.read())

File: cpp-recitations-solutions/helloworld/helloworld.cpp
#include <iostream>
#include <fstream>

int main(){
    //Say Hello to the World
    std::string hello;

    hello = "Hello Beautiful from C++";

    std::cout << hello << std::endl;
    std::cout<< "Version : "<< __cplusplus << std::endl;


    // BONUS : Write to a text file

    std::ofstream file("hellocpp.txt");
    file << hello;
    file.close();

    return 0; // tells system main function terminated successfully
}
